project/schema.go

```diff
@@ -0,0 +1,286 @@
+package project
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/docker/docker/pkg/nat"
+	"github.com/xeipuuv/gojsonschema"
+)
+
+var fieldsSchemaLoader gojsonschema.JSONLoader
+var serviceSchemaLoader gojsonschema.JSONLoader
```

Comment:
I don't think these need to be package level vars, they're only used in one function (plus a setup function, which could be made to return these instead).